lib/AST/Type.cpp

```diff
@@ -4065,12 +4065,10 @@ ReferenceCounting TypeBase::getReferenceCounting(
   CanType type = getCanonicalType();
   ASTContext &ctx = type->getASTContext();
 
-  // Determine which reference-counting scheme to use for an unknown object.
-  bool objCInterop = ctx.LangOpts.EnableObjCInterop;
-  auto getUnknownObjectReferenceCounting = [objCInterop] {
-    return objCInterop ? ReferenceCounting::Unknown
-                       : ReferenceCounting::Native;
-  };
+  // In the absence of Objective-C interoperability, everything uses native
+  // reference counting.
+  if (!ctx.LangOpts.EnableObjCInterop)
+    return ReferenceCounting::Native;
```

Comment:
No bridge objects?

Reply:
When we're not interoperating with Objective-C, the bridged classes are also defined in Swift, so it's still native everywhere.